INSTALL.md

```diff
@@ -101,3 +101,16 @@ If you've only changed `.c` or `.s` files, you can turn off the dependency scann
 
 `make NODEP=1`
 
+# Building with devkitARM's C compiler
+
+This project supports the `arm-none-eabi-gcc` compiler which ships with devkitARM r52.  To build this target, simply run:
+
+    make modern
+
+# Building with your own toolchain
+
+To build Pokemon Emerald with a toolchain other than devkitARM, override the `DEVKITARM` environment variable with the path to your toolchain.  Example:
+
+    make compare DEVKITARM=/usr/local/arm-none-eabi
```

Comment:
Overriding `DEVKITARM` doesn't sit well with me.  When you override it, it's no longer pointing to anything `DEVKITARM`-related, so I guess I would suggest defining a different environment variable.  I think it has potential to cause lots of confusion (in the Makefile, and users of the repo)